gke/README.md

```diff
@@ -54,9 +54,41 @@ gcloud auth login
 
 # Deploy esp service
 ./setup_esp.sh
+```
 
+1. Copy the `config.yaml` file into the `/deploy/gke` folder. Rename
+the file to describe the environment the clusters are being used for.
+
+> The filename used will be the `<ENV>` in subsequent commands. E.g. if you
+> named the yaml file `staging.yaml`, then the `ENV` below is `staging`.
+
+1. Go to the Google Cloud Console and make sure the following APIs are enabled.
```

Comment:
Can this step these 3x steps be automated in a script using the gcloud CLI?

Reply:
@juliawu can add these new roles in https://github.com/datacommonsorg/website/blob/master/gke/enable_services.sh

Reply:
Good idea, added to `enable_services.sh`